lib/foreman/renderer/scope/macros/helpers.rb

```diff
@@ -46,6 +46,23 @@ def to_json(data)
           def to_yaml(data)
             data.to_yaml
           end
+
+          apipie :method, "Generate a formatted time from a Ruby Time object or a Float object representing a UNIX timestamp" do
+            required :time, [Float, Time], desc: 'Ruby Time object or a Float object representing a UNIX timestamp'
+            optional :format, String, desc: 'Pattern to format time', default: '%Y-%-m-%-d %k:%M:%S %z'
+            optional :zone, String, desc: 'This parameter can be used for specify timezone of time, for example Europe/Prague', default: 'Default local timezone'
+            returns String
+          end
+          def format_time(time, format = '%Y-%-m-%-d %k:%M:%S %z', zone = Time.zone)
```

Comment:
Would it make sense to use kwarks instead?
```suggestion
          def format_time(time, format: '%Y-%-m-%-d %k:%M:%S %z', zone: Time.zone)
```
It would not force the argument order and it is bit more common pattern in macros

Reply:
I don't think that kwargs will be useful right know. There is only two optional parameters.